src/midcom/datamanager/template/form.php

```diff
@@ -417,7 +417,7 @@ public function form_label(FormView $view, array $data)
             return '';
         }
         if (!$data['label']) {
-            $data['label'] = $data['name'];
+            $data['label'] = $this->renderer->humanize($data['name']);
```

Comment:
This is redundant, `humanize` gets called again two lines below (also currently this might lead to double translation)